tests/soap/ProjectTest.php

```diff
@@ -201,7 +201,8 @@ protected function tearDown(): void {
 	 */
 	protected function getNewProjectName() {
 		do {
-			$t_name = $this->getName() . '_' . rand();
+			$t_name = ( method_exists( $this, 'getName' ) ?
```

Comment:
When there are version-specific code branches, I prefer to check based on version number as I did in my original, WIP commit. The reason is to facilitate future cleanup activities, to identify code that can be removed when we bump the minimum requirements and the check no longer applies.

A comment is also useful, e.g.
https://github.com/mantisbt/mantisbt/commit/7ccca0f9f991d692cd33156b48e2b90fc8bc0a45#diff-b52215be84fb8db024faf46fcfa124b282a4b141f1327588fc78660f3c36d980R201

Reply:
I think I've found a better way. Without depending on versions.